terraform/templates/local/docker_compose.tpl

```diff
@@ -8,6 +8,7 @@ services:
       - 55671:55671
 
   aws-ot-collector:
+    platform: linux/amd64
```

Comment:
is there any reason to make this configurable?

Reply:
This is to get the mock tests passing

Reply:
Our mocks only run on linux/amd64

Reply:
see https://github.com/aws-observability/aws-otel-collector/pull/870

Reply:
If you look in the logs it not defaulting the target which is weird because running a docker build will default